code/game/objects/items/stacks/sheets/sheet_types.dm

```diff
@@ -931,6 +931,23 @@ new /datum/stack_recipe("paper frame door", /obj/structure/mineral_door/paperfra
 /obj/item/stack/sheet/pizza/five
 	amount = 5
 
+/obj/item/stack/sheet/cheese
+	name = "cheese rectangles"
+	desc = "Man reinvents the wheel once more. The cheese wheel."
+	name = "cheese rectangle"
+	icon_state = "sheet-cheese"
+	mats_per_unit = list(/datum/material/cheese = SHEET_MATERIAL_AMOUNT)
+	merge_type = /obj/item/stack/sheet/cheese
+	material_type = /datum/material/cheese
+	material_modifier = 1
+
```

Comment:
You're probably going to want to add a `drop_sound` and `pickup_sound` for this. There's been issues with the parent "sheet" object playing a metal clang when touched, and I'm willing to guess this will affect cheese sheets.

Reply:
it will